bis-rules/package.json

```diff
@@ -68,10 +68,11 @@
     "extends": "plugin:@itwin/itwinjs-recommended",
     "rules": {
       "prefer-template": "off",
-      "dot-notation": "off",
+      "@typescript-eslint/dot-notation": "off",
+      "@typescript-eslint/no-misused-promises": "off",
```

Comment:
We should fix these issues rather than suppress this rule.  They often indicate real bugs

Reply:
Agreed!

Reply:
Agreed!